go.mod

```diff
@@ -27,33 +27,212 @@ require (
 	github.com/spf13/viper v1.8.1
 	github.com/stretchr/testify v1.7.0
 	github.com/thoas/go-funk v0.8.0
-	golang.org/x/mod v0.4.2
-	golang.org/x/tools v0.1.5
+	golang.org/x/mod v0.5.1
+	golang.org/x/tools v0.1.8
 	gomodules.xyz/jsonpatch/v3 v3.0.1
 	helm.sh/helm/v3 v3.6.2
-	k8s.io/api v0.22.2
-	k8s.io/apiextensions-apiserver v0.22.2
-	k8s.io/apimachinery v0.22.2
-	k8s.io/cli-runtime v0.21.0
-	k8s.io/client-go v0.22.2
-	k8s.io/kubectl v0.21.0
-	sigs.k8s.io/controller-runtime v0.10.0
-	sigs.k8s.io/controller-tools v0.7.0
-	sigs.k8s.io/kubebuilder/v3 v3.2.0
-	sigs.k8s.io/yaml v1.2.0
+	k8s.io/api v0.23.1
+	k8s.io/apiextensions-apiserver v0.23.1
+	k8s.io/apimachinery v0.23.1
+	k8s.io/cli-runtime v0.23.1
+	k8s.io/client-go v0.23.1
+	k8s.io/kubectl v0.23.1
+	sigs.k8s.io/controller-runtime v0.11.0
+	sigs.k8s.io/controller-tools v0.8.0
+	sigs.k8s.io/kubebuilder/v3 v3.3.0
+	sigs.k8s.io/yaml v1.3.0
 )
 
-replace (
-	github.com/Azure/go-autorest => github.com/Azure/go-autorest v14.2.0+incompatible // Required by OLM
```

Comment:
Those ones shows no longer required because we update the deps. 
See that the required and indirect ones cover these ones.